setup.py

```diff
@@ -5,7 +5,7 @@
 
 REPO_ROOT = path.abspath(path.dirname(__file__))
 
-VERSION = "2.2"
+VERSION = "3.0.0"
```

Comment:
Odd, the version had been 2.4, [then 3.0](https://github.com/octoenergy/xocto/commit/0f044c9641271c4c8a2da6213f4c6a575183d8ac#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7). How did it get moved down to 2.2?